apps/web/src/server/applications/case/examination-timetable/applications-timetable.controller.js

```diff
@@ -54,17 +54,21 @@ export const timetableTemplatesSchema = {
  * @type {import('@pins/express').RenderHandler<{}, {}, {}, {}, {caseId: string}>}
  */
 export async function viewApplicationsCaseExaminationTimeTable(request, response) {
-	let timetableItems = await getCaseTimetableItems(+request.params.caseId);
-	timetableItems = timetableItems.map((timetableItem) => {
+	const timetableItems = await getCaseTimetableItems(+request.params.caseId);
+	const timetableItemsViewData = timetableItems.map((timetableItem) => {
 		return {
 			...timetableItem,
 			description: JSON.parse(timetableItem.description),
+			startDate: timetableItem.startDate
+				? format(new Date(timetableItem.startDate), 'dd MMM yyyy')
```

Comment:
I see it was already in use, but we shouldn't use `format` directly as it won't handle time zones. `displayDate` in the nunjucks-filters folder might do what you need?